src/preset_cli/cli/superset/sync/dbt/databases.py

```diff
@@ -53,32 +48,36 @@ def sync_database(  # pylint: disable=too-many-locals, too-many-arguments
     if base_url and "external_url" not in meta:
         meta["external_url"] = str(base_url.with_fragment("!/overview"))
 
-    if import_db and databases:
-        _logger.info("Found an existing database connection, updating it")
-        database = databases[0]
-        meta.pop("uuid", None)
-
-        database = client.update_database(
-            database_id=database["id"],
-            database_name=database_name,
-            is_managed_externally=disallow_edits,
-            masked_encrypted_extra=connection_params.get("encrypted_extra"),
-            sqlalchemy_uri=connection_params["sqlalchemy_uri"],
-            **meta,
-        )
-
-        database["sqlalchemy_uri"] = connection_params["sqlalchemy_uri"]
-
-    elif import_db:
-        _logger.info("No database connection found, creating it")
-
-        database = client.create_database(
-            database_name=database_name,
-            is_managed_externally=disallow_edits,
-            masked_encrypted_extra=connection_params.get("encrypted_extra"),
-            **connection_params,
-            **meta,
-        )
+    if import_db:
+        if "connection_params" in meta:
+            connection_params = meta.pop("connection_params")
+        else:
+            connection_params = build_sqlalchemy_params(target)
```

Comment:
Just realized this can be done in a single line:
```suggestion
        connection_params = meta.pop(
            "connection_params",
            build_sqlalchemy_params(target),
        )
```